cdap-docs/_common/common-build.sh

```diff
@@ -355,7 +360,16 @@ function display_version() {
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
   echo "GIT_BRANCH_PARENT: ${GIT_BRANCH_PARENT}"
-  echo "GIT_BRANCH_CDAP_HYDRATOR: ${GIT_BRANCH_CDAP_HYDRATOR}"
+  echo "GIT_BRANCH_CASK_HYDRATOR: ${GIT_BRANCH_CASK_HYDRATOR}"
+  echo "CASK_HYDRATOR_VERSION: ${CASK_HYDRATOR_VERSION}"
+}
+
+function get_cask_hydrator_version() {
+  # $1 Branch of Hydrator Plugins to use
```

Comment:
The branch is used to download all of the Cask Hydrator documentation for that release...